### DISCUSSION TITLE
Discord event link updated 🏃

### DISCUSSION THREAD

**Discussion by AntonOsika** [AntonOsika/gpt-engineer#407] [Announcements]
I wanted to formally invite everyone who has only been here in GH Discussions – to Discord!!

[Discord](https://discord.gg/8tcDQ89Ej2)

Also a quick note:

Our 7 days discord link expired earlier today. Me and the forming community team apologize for the inconvenience.

It's up and you should be able to join it now though, and it's an open link until further notice.

**Comment by k1lgor** [comment]
I was able to join ✅

**Comment by Semsemq** [comment]
It's ok my dear
